integration/src/test/scala/com/snowplowanalytics/snowplow/collectors/scalastream/intergation/Containers.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0, and
+ * you may not use this file except in compliance with the Apache License
+ * Version 2.0.  You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the Apache License Version 2.0 is distributed on an "AS
+ * IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the Apache License Version 2.0 for the specific language
+ * governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.collectors.scalastream.intergation
+
+import cats.effect.{Resource, Sync}
+import com.dimafeng.testcontainers.GenericContainer
+import org.slf4j.LoggerFactory
+import org.testcontainers.containers.output.Slf4jLogConsumer
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.containers.{BindMode, Network, GenericContainer => JGenericContainer}
+import org.testcontainers.images.builder.ImageFromDockerfile
+
+object Containers {
+  private val network = Network.newNetwork()
+
+  def localstack: JGenericContainer[_] = {
+    val container = GenericContainer(
+      dockerImage  = "localstack/localstack-light:1.2.0",
+      exposedPorts = Seq(4566, 4567, 4568),
+      env          = Map("SERVICES" -> "kinesis", "DEFAULT_REGION" -> "eu-central-1", "USE_SSL" -> "1"),
+      waitStrategy = Wait.forLogMessage(".*AWS kinesis.CreateStream.*", 2),
+      fileSystemBind = Seq(
+        GenericContainer.FileSystemBind(
+          "./.localstack",
+          "/var/lib/localstack",
+          BindMode.READ_WRITE
+        ),
```

Comment:
It's a sort of temp dir for localstack. We're not currently using it in the tests so far. But I think it might come handy. It let's us have direct access to the mocked kinesis stream via the local filesystem rather than via a Kinesis client. If it ends up not being used, we can get rid of it; but for now I prefer to keep it around while we're still adding tests. It's helpful for debugging too.

Reply:
I added a comment to remove it if it's not used.